packages/truffle-core/test/mnemonic.js

```diff
@@ -1,19 +1,31 @@
 const assert = require("chai").assert;
+const sinon = require("sinon");
 const accountsInfo = require("../lib/mnemonics/mnemonic");
-const Config = require('truffle-config');
-const defaultUserConfig = Config.getUserConfig();
-
+const Configstore = require("configstore");
 
 describe("mnemonic", function() {
-  describe("#getOrGenerateMnemonic", function(){
+  beforeEach(() => {
+    sinon
+      .stub(Configstore.prototype, "get")
+      .returns(
+        "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
+      );
+    sinon.stub(Configstore.prototype, "set");
+  });
+  afterEach(() => {
+    Configstore.prototype.get.restore();
+    Configstore.prototype.set.restore();
+  });
+  describe("#getOrGenerateMnemonic", function() {
     it("checks user-level configuration for mnemonic and creates one if one is not present", function() {
       let mnemonic = accountsInfo.getOrGenerateMnemonic();
+      sinon.assert.calledOnce(Configstore.prototype.get);
       assert.exists(mnemonic);
       assert.isString(mnemonic);
-      assert.equal(mnemonic, defaultUserConfig.get('mnemonic'));
+      assert.equal(mnemonic, Configstore.prototype.get("mnemonic"));
```

Comment:
cc @fainashalts

Reply:
@eggplantzzz I'm confused why this code is reviewed in here -- was it changed somehow from my PR? Just want to be sure I understand. 
As for the stub, my thinking was that this test was hitting accountsInfo.getOrGenerateMnemonic(), which is not being stubbed. When that runs, it will hit Configstore within that function, which I have stubbed in this file. I am testing that it hits Configstore and that the value it receives through the function is the same as what I stubbed. Does that make sense?

Reply:
Oooh I see it was prettified? I thought that only happened if changes to the file were made?

Reply:
I pulled in your latest PR from `next` into this branch before updating the `pluggable-truffle` branch.